cli/resolver.rs

```diff
@@ -771,9 +763,7 @@ fn resolve_package_json_dep(
 #[derive(Debug)]
 pub struct WorkerCliNpmGraphResolver<'a> {
   npm_resolver: Option<&'a Arc<dyn CliNpmResolver>>,
-  // use a cache per worker so that another worker doesn't clear the
-  // cache of another worker
-  memory_cache: Rc<RefCell<HashMap<String, Arc<NpmPackageInfo>>>>,
```

Comment:
After some other refactors, this cache is only a bit of an optimization when many workers are doing npm resolution and the complexity is not worth keeping around for the edge case.